lib/tabula.ex

```diff
@@ -142,26 +145,33 @@ defmodule Tabula do
          cell when is_binary(cell) ->
            String.ljust(cell, w)
          cell when is_number(cell) ->
-           cell |> render_cell() |> String.rjust(w)
+           cell
+           |> render_cell()
+           |> String.rjust(w)
          cell ->
-           cell |> render_cell() |> String.ljust(w)
+           cell
+           |> render_cell()
+           |> String.ljust(w)
       end
     end)
   end
 
   defp spacers(widths, opts) do
-    map(widths, &(Stream.repeatedly(fn -> style(:spacer, opts) end))
+    map widths, &(Stream.repeatedly(fn -> style(:spacer, opts) end)
                 |> Stream.take(&1)
                 |> join)
```

Comment:
Perhaps it would be better to move this to a separate private function or at least bind it to a meaningful name before using in `map`?